### PR TITLE
UnrealBloomPass now is transparent~

### PR DESCRIPTION
**Pull request by moshuying**
**Description**

UnrealBloomPass now is transparent~ But my ability is limited. After I was transparent, I found that the effect in example/webgl_postprocessing_unreal_bloom.html seemed to be slightly different from the previous one. I tried my best to make it better, but it can only be done like this...

image

image

>Hope someone can make it better~

### CODE REVIEW

**Comment by moshuying** [comment]
Can't pass the test temporarily because of the updated post-processing effect?

File: examples/jsm/postprocessing/UnrealBloomPass.js [hunk 1]
```diff
@@ -23,7 +24,7 @@ import { LuminosityHighPassShader } from '../shaders/LuminosityHighPassShader.js
  * Reference:
  * - https://docs.unrealengine.com/latest/INT/Engine/Rendering/PostProcessEffects/Bloom/
  */
-class UnrealBloomPass extends Pass {
+class UnrealBloomPass extends MaskPass {
```
Comment (Mugen87) [inline examples/jsm/postprocessing/UnrealBloomPass.js:27]: Why are you deriving of `MaskPass`?

**Comment by Mugen87** [comment]
> Can't pass the test temporarily because of the updated post-processing effect?

When you change the visuals of an example, you have to regenerate its screenshot via:
```
npm run make-screenshot <example-name>
```

**Comment by Mugen87** [comment]
> UnrealBloomPass now is transparent

What is your motivation for this change? Is it to fix #14104? 

The linked issue is about setting `alpha: true` when creating the renderer? Does this actually work? I don't see that you have updated one of the bloom examples accordingly.

**Comment by moshuying** [comment]
> The linked issue is about setting `alpha: true` when creating the renderer. Does the pass support this setup now? I don't see that you have updated one of the bloom examples accordingly.

Thank you, but I didn't know about this issue before I did this. In the case I tried, it has nothing to do with render turning on transparent rendering. Whether or not transparent rendering is turned on, it will not affect the final rendering effect.

**Comment by Mugen87** [comment]
I have hard time to understand what you are trying to solve with this PR. The implementation is not clear to me.

Does it mean the scene's background is now not affected by bloom anymore?

If so, I suggest to use the workaround from https://github.com/mrdoob/three.js/issues/14104#issuecomment-1088395025.

**Comment by Mugen87** [comment]
Closing due to lack of feedback.

**Pull request closed without merging**